modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -44,22 +44,24 @@ const Container = styled(Box)<StyledType>({
 // positioning, but seems to when using flexbox centering. This messes up Popper calculations inside
 // the Modal. The centering container forces a "center" pixel calculation by making sure the width
 // is always an even number
-const CenteringContainer = styled('div')({
+const ResponsiveCenteringContainer = styled('div')(({theme}) => ({
```

Comment:
What do you think about rename to just `ResponsiveCenteringContainer`? It seems that Responsive is more important here for meaning.

Reply:
oo can you clarify? I think the change shows that I did rename it to `ResponsiveCenteringContainer` ? Or are you saying to not add the responsive piece?

Reply:
I meant to rename to `ResponsiveContainer`.